pages/user_not_found_in_adr.js

```diff
@@ -2,16 +2,35 @@ import { t } from "@lingui/macro";
 import Link from "next/link";
 import { ButtonGroup, Button } from "react-bootstrap";
 import { LogInLayout } from "@/components/Layout";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowsRotate } from "@fortawesome/free-solid-svg-icons";
 
 export default function UserNotFoundInAdrPage() {
   return (
     <LogInLayout>
       <h3>{t`Current user has not been found in ADR.`}</h3>
       <hr />
-      <p>{t`Your ADR account has not yet been used to log into ADR. Please log into ADR and come back here.`}</p>
-        <Link href={process.env.NEXT_PUBLIC_CKAN_SITE_URL + "/user/saml2login"}>
-          <Button variant="outline-danger">{t`Log into ADR`}</Button>
+      <p>{t`Your SSO account has not yet been used to log in to the ADR. Please use the "Go to ADR" button below to
+      visit ADR and get your account created.
+       Then come back here and click "Refresh Page".`}</p>
```

Comment:
Since the error message is changed we need to update locales/{en,fr,pt}/messages.po accordingly.